pallets/capacity/src/lib.rs

```diff
@@ -465,35 +482,39 @@ impl<T: Config> Pallet<T> {
 		let mut capacity_details = Self::get_capacity_for(target).unwrap_or_default();
 		capacity_details.deposit(&amount, &capacity).ok_or(ArithmeticError::Overflow)?;
 
-		Self::set_staking_account(&staker, staking_account);
+		Self::set_staking_account(&staker, staking_account)?;
```

Comment:
Since we can fail here we can probably add some new unit tests around this.

Reply:
This code is now outdated. I will have a look at the new code and see if any new tests are still needed.